modules/images/webp-uploads/readme.txt

```diff
@@ -0,0 +1,63 @@
+=== Performance Lab - WebP Uploads Module ===
```

Comment:
The plugin should be called like the module, "WebP Uploads". It does not require Performance Lab, so shouldn't look as if it was an extension.